---
layout: default
---


## Next meeting

**TBD**  
Zoom meeting (details will be sent via email)


## About the club

This journal club is hosted by [Melbourne Integrative Genomics
(MIG)](http://research.unimelb.edu.au/integrative-genomics) at the University
of Melbourne.  It is currently managed by [Damjan
Vukcevic](http://research.unimelb.edu.au/integrative-genomics/research/statistical-genetics-vukcevic).


## Mailing list

To be notified about future meetings of the club, please
[subscribe](https://lists.unimelb.edu.au/subscribe/statgen) to the mailing
list.  Membership is open to anyone who is interested.

To manage your subscription or find out more about the list, please visit the
list's [website](https://lists.unimelb.edu.au/info/statgen).


## Proposed papers

*Please send suggestions to
[Damjan Vukcevic](mailto:damjan.vukcevic@unimelb.edu.au).
Papers must be related to genetics or genomics and must have substantial
statistical content.*


Liu X, Li YI, Pritchard JK (2019).  [Trans effects on gene expression can drive
omnigenic inheritance](https://doi.org/10.1016/j.cell.2019.04.014), *Cell*.
([pubmed](https://www.ncbi.nlm.nih.gov/pubmed/31051098),
[biorxiv](https://doi.org/10.1101/425108))

Wainschtein P, ..., Weir BS, Laurie CC, Yang J, Visscher PM (2019).  [Recovery
of trait heritability from whole genome sequence
data](https://doi.org/10.1101/588020), *bioRxiv*.

Gigante S, van Dijk D, Moon K, Strzalkowski A, Wolf G, Krishnaswamy S (2019).
[Modeling Global Dynamics from Local Snapshots with Deep Generative Neural
Networks](https://arxiv.org/abs/1802.03497), *arXiv*.

Wald NJ, Old R (2019).  [The illusion of polygenic disease risk
prediction](https://doi.org/10.1038/s41436-018-0418-5), *Genetics in Medicine*.
([pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30635622))

Martin AR, Kanai M, Kamatani Y, Okada Y, Neale BM, Daly MJ (2019).  [Clinical
use of current polygenic risk scores may exacerbate health
disparities](https://doi.org/10.1038/s41588-019-0379-x), *Nature genetics*.
([pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30926966))

Speidel L, Forest M, Shi S, Myers S (2019). [A method for genome-wide genealogy
estimation for thousands of samples](https://doi.org/10.1101/550558), bioRxiv.
([software](https://myersgroup.github.io/relate/))

Hou K, Burch KS, Majumdar A, Shi H, N Mancuso N, Wu Y, Sankararaman S, Pasaniuc
B (2019).  [Accurate estimation of SNP-heritability from biobank-scale data
irrespective of genetic architecture](https://doi.org/10.1101/526855),
*bioRxiv*.

Zhu X, Stephens M (2018).  [Large-scale genome-wide enrichment analyses
identify new trait-associated genes and pathways across 31 human
phenotypes](https://doi.org/10.1038/s41467-018-06805-x), *Nature
Communications*. ([pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30341297),
[pmc](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6195536/))

Zhang Y, Qi G, Park JH, Chatterjee N (2018).  [Estimation of complex
effect-size distributions using summary-level statistics from genome-wide
association studies across 32 complex
traits](https://doi.org/10.1038/s41588-018-0193-x), *Nature Genetics*.
([pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30104760),
[biorxiv](https://doi.org/10.1101/175406))

Márquez-Luna C, Gazal S, Loh P-R, Kim SS, Furlotte N, Auton A, 23andMe, Price
AL (2018).  [Modeling functional enrichment improves polygenic prediction
accuracy in UK Biobank and 23andMe data sets](https://doi.org/10.1101/375337),
*bioRxiv*.

Zhu Z, ...,  Visscher PM, Wray NR, Yang J (2018).  [Causal associations between
risk factors and common diseases inferred from GWAS summary
data](https://doi.org/10.1038/s41467-017-02317-2), *Nature Communications*.
([pubmed](https://www.ncbi.nlm.nih.gov/pubmed/29335400),
[pmc](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5768719/),
[biorxiv](https://doi.org/10.1101/168674))

Brown BC, AGEN-T2G, Ye CJ, Price AL, Zaitlen N (2016).  [Transethnic
Genetic-Correlation Estimates from Summary
Statistics](https://doi.org/10.1016/j.ajhg.2016.05.001), *AJHG*.
([pubmed](https://www.ncbi.nlm.nih.gov/pubmed/27321947),
[pmc](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5005434/),
[biorxiv](https://doi.org/10.1101/036657))


## Previous papers

**12 Apr 2021**  Cherryh C, Minh BQ, Lanfear R (2021).  [A test statistic to
quantify treelikeness in
phylogenetics](https://doi.org/10.1101/2021.02.16.431544), *bioRxiv*.

**29 Mar 2021.**  Speidel L, Cassidy L, Davies RW, Hellenthal G, Skoglund P,
Myers S (2021). [Inferring population histories for ancient genomes using
genome-wide genealogies](https://doi.org/10.1101/2021.02.17.431573), *bioRxiv*.

**15 Mar 2021.**  Wohns AW, Wong Y, Jeffery B, Akbari A, Mallick S, Pinhasi R,
Patterson N, Reich D, Kelleher J, McVean G (2021).  [A unified genealogy of
modern and ancient genomes](https://doi.org/10.1101/2021.02.16.431497),
*bioRxiv*.

**10 Dec 2019.**  Barton N, Hermisson J, Nordborg M (2019).  [Why structure
matters](https://doi.org/10.7554/eLife.45380), *eLife*.
([pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30895925),
[pmc](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6428565/))

**24 Sep 2019.**  Fortes-Lima CA, Laurent R, Thouzeau V, Toupance B, Verdu
P (2019).  [Complex genetic admixture histories reconstructed with Approximate
Bayesian Computations](https://doi.org/10.1101/761452), *bioRxiv*.

**13 Sep 2019.**  Almarri MA, Bergström A, Prado-Martinez J, Dunham AS, Chen Y,
Tyler-Smith C, Xue Y (2019).  [Population Structure, Stratification and
Introgression of Human Structural Variation in the
HGDP](https://doi.org/10.1101/746172), *bioRxiv*.

**27 Aug 2019.**  Veturi Y, de Los Campos G, Yi N, Huang W, Vazquez AI, Kühnel
B (2019). [Modeling Heterogeneity in the Genetic Architecture of Ethnically
Diverse Groups Using Random Effect Interaction
Models](https://doi.org/10.1534/genetics.119.301909), *Genetics*.
([pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30796011),
[pmc](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6456318/))

**22 Jul 2019.**  Urbut SM, Wang G, Carbonetto P, Stephens M (2019).  [Flexible
statistical methods for estimating and testing effects in genomic studies with
multiple conditions](https://doi.org/10.1038/s41588-018-0268-8), *Nature
Genetics*. ([pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30478440),
[biorxiv](https://doi.org/10.1101/096552))

**27 May 2019.**  Ge T, Chen CY, Ni Y, Feng, YA, Smoller JW (2019).  [Polygenic
prediction via Bayesian regression and continuous shrinkage
priors](http://dx.doi.org/10.1038/s41467-019-09718-5), *Nature Communications*.
([biorxiv](https://doi.org/10.1101/416859))

**2 May 2019.**  Karczewski KJ, ..., Neale BM, Daly MJ, MacArthur DG (2019).
[Variation across 141,456 human exomes and genomes reveals the spectrum of
loss-of-function intolerance across human protein-coding
genes](https://doi.org/10.1101/531210), *bioRxiv*.
([blog](http://www.cureffi.org/2019/01/28/using-human-lof-variants-to-evaluate-drug-targets/))

**11 Apr 2019.**  Albers PK, McVean G (2018).  [Dating genomic variants and
shared ancestry in population-scale sequencing
data](https://doi.org/10.1101/416610), *bioRxiv*.

**11 Mar 2019.**  Zhang C, Matsen IV FA (2018).  [Generalizing Tree Probability Estimation via Bayesian Networks](https://papers.nips.cc/paper/7418-generalizing-tree-probability-estimation-via-bayesian-networks), NIPS 2018. ([pdf](https://papers.nips.cc/paper/7418-generalizing-tree-probability-estimation-via-bayesian-networks.pdf), [blog](https://matsen.fredhutch.org/general/2018/12/05/sbn.html))

**10 Dec 2018.**  Khera AV, Chaffin M, Aragam KG, Haas ME, Roselli C, Choi SH,
Natarajan P, Lander ES, Lubitz SA, Ellinor PT, Kathiresan S (2018).
[Genome-wide polygenic scores for common diseases identify individuals with
risk equivalent to monogenic
mutations](https://doi.org/10.1038/s41588-018-0183-z), *Nature Genetics*.
([pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30104762))

**19 Nov 2018.**  Kelleher J, Wong Y, Albers P, Wohns AW, McVean G (2018).
[Inferring the ancestry of everyone](https://doi.org/10.1101/458067),
*bioRxiv*.

**5 Nov 2018.**  Bradburd GS, Coop GM, Ralph PL (2018).  [Inferring Continuous
and Discrete Population Genetic Structure Across
Space](https://doi.org/10.1534/genetics.118.301333), *Genetics*.
([pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30026187),
[pmc](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6116973/),
[biorxiv](https://doi.org/10.1101/189688))

**8 Oct 2018.**  Shi H, Kichaev G, Pasaniuc B (2016).  [Contrasting the Genetic
Architecture of 30 Complex Traits from Summary Association
Data](https://doi.org/10.1016/j.ajhg.2016.05.013), *AJHG*.
([pubmed](https://www.ncbi.nlm.nih.gov/pubmed/27346688),
[pmc](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5005444/),
[biorxiv](https://doi.org/10.1101/035907))

**10 Sep 2018.**  Tran D, Blei DM (2017).  [Implicit Causal Models for
Genome-wide Association Studies](https://arxiv.org/abs/1710.10742), *arXiv*.
([openreview](https://openreview.net/forum?id=SyELrEeAb))

**13 Aug 2018.**  Mak TSH, Porsch RM, Choi SW, Zhou X, Sham PC (2017).
[Polygenic scores via penalized regression on summary
statistics](https://doi.org/10.1002/gepi.22050), *Genetic Epidemiology*.

**18 Jul 2018.**  He Q, Pilosof S, Tiedje KE, Ruybal-Pesántez S, Artzy-Randrup
Y, Baskerville EB, Day KP, Pascual M (2018).  [Networks of genetic similarity
reveal non-neutral processes shape strain structure in *Plasmodium
falciparum*](https://doi.org/10.1038/s41467-018-04219-3), *Nature
Communications*. ([pubmed](https://www.ncbi.nlm.nih.gov/pubmed/29739937),
['behind the
paper'](https://naturemicrobiologycommunity.nature.com/users/108152-qixin-he/posts/32988-why-a-diverse-approach-is-needed-to-tackle-a-hyperdiverse-malaria-antigen-system))

**27 Jun 2018.**  Durvasula A, Sankararaman S (2018).  [Recovering signals of
ghost archaic admixture in the genomes of present-day
Africans](https://doi.org/10.1101/285734), *bioRxiv*.

**28 May 2018.**  Kelleher J, Thornton K, Ashander J, Ralph P (2018).
[Efficient pedigree recording for fast population genetics
simulation](https://doi.org/10.1101/248500), *bioRxiv*.

**9 May 2018.**  Cabreros I, Storey JD (2017).  [A nonparametric estimator of
population structure unifying admixture models and principal components
analysis](https://doi.org/10.1101/240812), *bioRxiv*.

**18 Apr 2018.**  Chan J, Perrone V, Spence JP, Jenkins PA, Mathieson S, Song
YS (2018).  [A Likelihood-Free Inference Framework for Population Genetic Data
using Exchangeable Neural Networks](https://doi.org/10.1101/267211 ),
*bioRxiv*. ([arxiv](https://arxiv.org/abs/1802.06153))

**28 Mar 2018.**  Li YI, Knowles DA, Humphrey J, Barbeira AN, Dickinson SP, Im
HK, Pritchard JK (2018).  [Annotation-free quantification of RNA splicing using
LeafCutter](https://dx.doi.org/10.1038/s41588-017-0004-9), *Nature Genetics*.
([biorxiv](https://doi.org/10.1101/044107))
